src/containers/shared/utils.js

```diff
@@ -8,6 +8,8 @@ const BILLION = MILLION * THOUSAND
 const TRILLION = BILLION * THOUSAND
 const QUADRILLION = TRILLION * THOUSAND
 
+export const TRADING_FEE_TOTAL = 1000
```

Comment:
Is this constant used at all externally, now that the `formatTradingFee` method exists?